<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>218</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>218</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>218</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Patch Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/resources/PGen.png</normaloff>:/resources/PGen.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="mGroupFrom">
     <property name="title">
      <string>From</string>
     </property>
     <layout class="QHBoxLayout" name="mGroupFromLayout">
      <item>
       <widget class="LineEdit" name="mFromLineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="mFromButtonZip">
        <property name="text">
         <string>zip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="mFromButtonDir">
        <property name="text">
         <string>dir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGroupTo">
     <property name="title">
      <string>To</string>
     </property>
     <layout class="QHBoxLayout" name="mGroupToLayout">
      <item>
       <widget class="LineEdit" name="mToLineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="mToButtonZip">
        <property name="text">
         <string>zip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="mToButtonDir">
        <property name="text">
         <string>dir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGroupSaveAs">
     <property name="title">
      <string>Save As</string>
     </property>
     <layout class="QHBoxLayout" name="mGroupSaveAsLayout">
      <item>
       <widget class="LineEdit" name="mSaveAsLineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="mSaveAsButton">
        <property name="text">
         <string>zip</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mStart">
     <property name="text">
      <string>Generate</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
